kfs-core/src/main/java/edu/arizona/kfs/fp/document/ProcurementCardDocument.java

```diff
@@ -32,6 +43,55 @@ public ProcurementCardHolder getProcurementCardHolder() {
     public void setProcurementCardHolder(ProcurementCardHolder procurementCardHolder) {
         this.procurementCardHolder = procurementCardHolder;
     }
+    
+    /**
+     * @return the previous fiscal year used with all GLPE
+     */
+    protected static final Integer getPreviousFiscalYear() {
+        int i = SpringContext.getBean(UniversityDateService.class).getCurrentFiscalYear().intValue() - 1;
+        return new Integer(i);
+    }
+
+    @Override
+    public void customizeExplicitGeneralLedgerPendingEntry(GeneralLedgerPendingEntrySourceDetail postable, GeneralLedgerPendingEntry explicitEntry) {
+        Date temp = getProcurementCardTransactionPostingDetailDate();
+        
+        if( temp != null && allowBackpost(temp) ) {
+            Integer prevFiscYr = getPreviousFiscalYear();
+            
+            explicitEntry.setUniversityFiscalPeriodCode(FINAL_ACCOUNTING_PERIOD);
+            explicitEntry.setUniversityFiscalYear(prevFiscYr);
+            
+            if( !getDocumentHeader().getDocumentDescription().contains("FY " + prevFiscYr) ) {
```

Comment:
Fixed. As for the documentDescription, Misty said the value is  the credit card last four digits and card name, which is being set in the createAndValidateAccountingLines method of (edu/arizona)ProcurementCardCreateDocumentServiceImpl.java.